scripts/systemd/gogs.service

```diff
@@ -8,19 +9,43 @@ After=network.target
 #After=redis.service
 
 [Service]
-# Modify these two values and uncomment them if you have
-# repos with lots of files and get an HTTP error 500 because
-# of that
-###
-#LimitMEMLOCK=infinity
-#LimitNOFILE=65535
-Type=simple
+Restart=on-failure
+
+; User and group the process will run as.
 User=git
 Group=git
+ 
+; Gogs directoryies.
+Environment=USER=git HOME=/home/git
 WorkingDirectory=/home/git/gogs
+ 
 ExecStart=/home/git/gogs/gogs web
-Restart=always
-Environment=USER=git HOME=/home/git
+
+; Repos with lots of files and get an HTTP error 500 will need that
+;LimitMEMLOCK=infinity
```

Comment:
Again, why did you change things without changing them? (same for line above and below)